### DISCUSSION TITLE
7.9 일일 회의

### DISCUSSION THREAD

**Discussion by jlee38266** [3sam5oh/webtoon-recommendation-service#30] [일일 회의]
금일 진행 점검

전체
- #14 #16
- #13 #15 
- #22 

김아영
- #20 

이재원
- #21
- #19 

이준형
- #20 

정태원
- #23 

7.11 산출물 요구사항
![image](https://github.com/3sam5oh/webtoon-recommendation-service/assets/152199695/c1fa26ef-3225-47e7-b56e-80a9fc40e18b)
![image](https://github.com/3sam5oh/webtoon-recommendation-service/assets/152199695/e2ca2df7-035e-4c2c-9317-1cd862007b7a)

**Comment by joonangler** [comment]
## 역할 분배 예시 (대분류)
- 테스트
- 인프라
- 데이터 관련 스키마
- 스케일링 팩터
